tests/test_unsafe.py

```diff
@@ -0,0 +1,374 @@
+from contextlib import nullcontext as does_not_raise
+from typing import ContextManager
+
+import pytest
+import yaml
+from jinja2.ext import Extension
+from plumbum import local
+from plumbum.cmd import git
+
+from copier.cli import CopierApp
+from copier.errors import UserMessageError
+from copier.main import run_copy, run_update
+from copier.types import AnyByStrDict
+
+from .helpers import build_file_tree
+
+
+class JinjaExtension(Extension):
+    ...
+
+
+@pytest.mark.parametrize(
+    ("spec", "expected"),
+    [
+        (
+            {},
+            does_not_raise(),
+        ),
+        (
+            {"_tasks": []},
+            does_not_raise(),
+        ),
+        (
+            {"_tasks": ["touch task.txt"]},
+            pytest.raises(
+                UserMessageError, match="Template uses unsafe feature: tasks"
+            ),
+        ),
+        (
+            {"_migrations": []},
+            does_not_raise(),
+        ),
+        (
+            {
+                "_migrations": [
+                    {
+                        "version": "v1",
+                        "before": ["touch v1-before.txt"],
+                        "after": ["touch v1-after.txt"],
+                    }
+                ]
+            },
+            does_not_raise(),
+        ),
```

Comment:
Why this case doesn't raise? 🤔

Reply:
Because `run_copy()` doesn't execute migration tasks, so I think they should be ignored by the check.